lib/bot-runner.rb

```diff
@@ -4,8 +4,15 @@ def initialize(client)
     @client = client
   end
 
-  def say_dumb_things()
-    @client.send('i liek gaem')
+  def say(s)
```

Comment:
This is terrible, it goes say -> send -> update but they all do the same thing. That said, I'm not sure I want them all to have the same name. I might use 'send' for both of the internal ones, since I can't change the tweeterer API. That said, it'd be cool to make it so it could also be used as a slack bot or whatever, and the bot takes a client object of any kind as long as it has a wrapper with a 'send' method.